test/e2e-test-application/src/luigi-config/extended/userSettings.js

```diff
@@ -38,33 +66,6 @@ class UserSettings {
         }
       }
     },
-    language: {
```

Comment:
Please change it back. First item in the list should be `User Account` and second `Language` otherwise we will get a few failing tests + more failing backwards compatibility tests.